e2e/node/e2e.test.ts

```diff
@@ -102,186 +96,140 @@ const invalidCredentialClaims = {
       "@type": "@id",
     },
   },
-  inbox: "https://pod.inrupt.com/solid-client-e2e-tester-ess/inbox/",
 };
 
-type OidcIssuer = string;
-type VcService = string;
-type VcSubject = string;
-type ClientId = string;
-type ClientSecret = string;
-type AuthDetails = [OidcIssuer, ClientId, ClientSecret, VcService, VcSubject];
-// Instructions for obtaining these credentials can be found here:
-// https://github.com/inrupt/solid-client-authn-js/blob/1a97ef79057941d8ac4dc328fff18333eaaeb5d1/packages/node/example/bootstrappedApp/README.md
-const serversUnderTest: AuthDetails[] = [
-  // Note: Disabled due to PodSpaces 2.0 migration:
-  // pod.inrupt.com:
-
-  // dev-next.inrupt.com:
-  [
-    // Cumbersome workaround, but:
-    // Trim `https://` from the start of these URLs,
-    // so that GitHub Actions doesn't replace them with *** in the logs.
-    process.env.E2E_TEST_DEV_NEXT_IDP_URL!.replace(/^https:\/\//, ""),
-    process.env.E2E_TEST_DEV_NEXT_VC_SERVICE!.replace(/^https:\/\//, ""),
-    process.env.E2E_TEST_DEV_NEXT_VC_SUBJECT!.replace(/^https:\/\//, ""),
-    process.env.E2E_TEST_DEV_NEXT_CLIENT_ID!,
-    process.env.E2E_TEST_DEV_NEXT_CLIENT_SECRET!,
-  ],
-];
-
-describe.each(serversUnderTest)(
-  "VC client end-to-end tests authenticated to [%s], issuing from [%s] for [%s]",
-  (
-    oidcIssuerDisplay,
-    vcServiceDisplay,
-    vcSubjectDisplay,
-    clientId,
-    clientSecret
-  ) => {
-    const oidcIssuer = new URL(`https://${oidcIssuerDisplay}`).href;
-    const vcService = new URL(`https://${vcServiceDisplay}`).href;
-    const vcSubject = new URL(`https://${vcSubjectDisplay}`).href;
-
-    it("has the appropriate environment variables", () => {
-      expect(oidcIssuer).toBeDefined();
-      expect(clientId).toBeDefined();
-      expect(clientSecret).toBeDefined();
-      expect(vcService).toBeDefined();
-      expect(vcSubject).toBeDefined();
-    });
-
-    const session = new Session();
-    // let vcConfiguration: Partial<{
-    //   derivationService: string;
-    //   issuerService: string;
-    //   statusService: string;
-    //   verifierService: string;
-    // }>;
-
-    beforeEach(async () => {
-      await session.login({
-        oidcIssuer,
-        clientId,
-        clientSecret,
-      });
-      // The following code snippet doesn't work in Jest, probably because of
-      // https://github.com/standard-things/esm/issues/706 which seems to be
-      // related to https://github.com/facebook/jest/issues/9430. The JSON-LD
-      // module depends on @digitalbazaar/http-client, which uses esm, which
-      // looks like it confuses Jest. Working around this by hard-coding the
-      // endpoints IRIs.
-      // vcConfiguration = await getVerifiableCredentialApiConfiguration(
-      //   vcService
-      // );
-    });
+const {
+  environment,
+  idp: oidcIssuer,
+  vcProvider,
+  clientCredentials,
+} = getNodeTestingEnvironment({
```

Comment:
Would it be better to avoid the destructuring and just use `const env = getNodeTestingEnvironment(` ?